src/RVRP_data.jl

```diff
@@ -96,16 +112,17 @@ struct RvrpInstance
     travel_distance_matrix::Array{Float64,2}
     travel_time_matrix::Array{Float64,2}
     depots::Vector{Depot}
-    # Requests
-    pickups::Vector{Pickup}
-    deliveries::Vector{Delivery}
-    shipments::Vector{Shipment}
+    pickup_points::Vector{PickupPoints}
+    delivery_points::Vector{DeliveryPoints}
+    products::Vector{Product}
```

Comment:
Typo: Elements of these vectors need to be SpecificProduct and ProductCategory

Reply:
i am already fixing these things, there were others too